src/NuGet.Clients/VisualStudio.Facade/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "NuGet.Common": "3.6.0-*",
```

Comment:
>Common [](start = 11, length = 6)

How have we avoid this dependency until now? Are all of the facade types fully declared in this assembly up until now?